modules/op-query-results-name.adoc

```diff
@@ -0,0 +1,81 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="query-results-uuid_{context}"]
+= Querying for results and records by name
+
+[role="_abstract"]
+You can list and query results and records using their names.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility and prepared its environment to quety {tekton-results}.
+* You installed the link:https://pypi.org/project/yq/[`yq` utility].
```

Comment:
Why does a user need to install additional software that Red Hat doesn't distribute to use `pipelines`?